src/GCHP_GridComp/GCHPctmEnv_GridComp/GCHPctmEnv_GridCompMod.F90

```diff
@@ -1,668 +1,460 @@
+                              LONG_NAME='pressure_weighted_accumulated_yward_mass_flux', &
+                              UNITS='Pa m+2 s-1', &
+                              PRECISION=ESMF_KIND_R8, &
+                              DIMS=MAPL_DimsHorzVert, &
+                              VLOCATION=MAPL_VLocationCenter, &
+                              RC=STATUS)
+      _VERIFY(STATUS)
+      call MAPL_AddExportSpec(gc, &
+                              SHORT_NAME='UpwardsMassFlux', &
+                              LONG_NAME='upward_mass_flux_of_air', &
+                              UNITS='kg m-2 s-1', &
+                              PRECISION=ESMF_KIND_R8, &
+                              DIMS=MAPL_DimsHorzVert, &
+                              VLOCATION=MAPL_VLocationEdge, &
+                              RC=STATUS)
       _VERIFY(STATUS)
-
+      
+      call lgr%debug('Adding timers')
+      call MAPL_TimerAdd(gc, name="INITIALIZE", RC=STATUS)
+      _VERIFY(STATUS)
+      call MAPL_TimerAdd(gc, name="RUN", RC=STATUS)
+      _VERIFY(STATUS)
+      
+      call lgr%debug('Calling MAPL_GenericSetServices')
+      call MAPL_GenericSetServices(gc, RC=STATUS)
+      _VERIFY(STATUS)
+      
       _RETURN(ESMF_SUCCESS)
-  
-      end subroutine SetServices
-!
-!EOC
-!-------------------------------------------------------------------------
-!BOP
-!
-! !IROUTINE: Initialize -- Initialized method for composite the CTMder
-!
-! !INTERFACE:
-!
-      subroutine Initialize ( GC, IMPORT, EXPORT, CLOCK, RC )
-!
-! !INPUT/OUTPUT PARAMETERS:
+      
+   end subroutine SetServices
+
+
+   subroutine Initialize(GC, IMPORT, EXPORT, CLOCK, RC)
       type(ESMF_GridComp), intent(inout) :: GC     ! Gridded component 
       type(ESMF_State),    intent(inout) :: IMPORT ! Import state
       type(ESMF_State),    intent(inout) :: EXPORT ! Export state
       type(ESMF_Clock),    intent(inout) :: CLOCK  ! The clock
-!
-! !OUTPUT VARIABLES:
-      integer, optional,   intent(  out) :: RC     ! Error code
-!
-! !DESCRIPTION: 
-!  The Initialize method of the CTM Derived Gridded Component.
-!
-!EOP
-!-------------------------------------------------------------------------
-!BOC
-!
-! !LOCAL VARIABLES:
+      integer, optional,   intent(out)   :: RC     ! Error code
+
+      ! Locals
       __Iam__('Initialize')
-      character(len=ESMF_MAXSTR)    :: COMP_NAME
+      character(len=ESMF_MAXSTR) :: COMP_NAME
       REAL, POINTER, DIMENSION(:,:) :: cellArea
-      type(ESMF_Grid)               :: esmfGrid
-      type (ESMF_VM)                :: VM
-      integer                       :: im, jm, km, i
-      type(MAPL_MetaComp), pointer  :: ggState      ! GEOS Generic State
-      type (ESMF_Config)            :: CF
-      integer                       :: dims(3)
+      type(ESMF_Grid) :: esmfGrid
+      type (ESMF_VM) :: VM
+      type(MAPL_MetaComp), pointer :: ggState      ! GEOS Generic State
+      type (ESMF_Config) :: CF
```

Comment:
Having the :: lined up has better readability.

Reply:
(See my comment in the main thread)